drivers/ma_yya_report/app/models/ma_yya_report/universe_calculator.rb

```diff
@@ -147,15 +147,8 @@ def calculate(&post_processor)
     end
 
     private def race_code
-      {
-        'AmIndAKNative' => 1,
-        'Asian' => 2,
-        'BlackAfAmerican' => 3,
-        'NativeHIOtherPacific' => 4,
-        'White' => 5,
-        'HispanicLatinaeo' => 6,
-        'MidEastNAfrican' => 7,
-      }.freeze
+      # NativeHIOtherPacific => NativeHIPacific -- ok?
+      HudUtility2024.race_id_to_field_name.excluding(8, 9, 99).invert
```

Comment:
Is this correct? one value changed as noted in the comment

Reply:
That look like a bug fix to me.  This gets called via the `race` method and is looking at a destination client, so should be using the new columns.